.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
       - opendistro-*
 
 env:
-  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3
+  GRADLE_OPTS: -Dhttp.keepAlive=false -DCI=true
```

Comment:
Generally wary of things that behave different under CI. Looks like this changes the number of retries for tests? I would prefer if this were more explicit, e.g. `-Dtests.retry=3`.

Reply:
https://github.com/opensearch-project/security/pull/1592#discussion_r804884915